docs/RELEASING.md

```diff
@@ -93,4 +93,27 @@
 
 ### Create release on Google Play
 
-TODO
+1. Go to the [Google Play Console](https://play.google.com/console/)
+2. Select the _K-9 Mail_ app
+3. Click on _Open testing_ in the left sidebar
+4. Click on _Create new release_
+5. Upload the APK to _App bundles_
+6. Fill out Release name (e.g. "$versionCode ($versionName)")
+7. Fill out Release notes (copy from `app-k9mail/fastlane/metadata/android/en-US/changelogs/${versionCode}.txt`)
+8. Click _Next_
+9. Review the release
+10. Start with a staged rollout (usually 20%)
```

Comment:
I think we should roll out beta releases to 100% of beta testers at once. We want the beta to hit as many devices as fast as possible.